### ISSUE TITLE
Unknown event handler property

### ISSUE THREAD

**Issue by gavinbelson** [rcdexta/react-trello#164]
Hey,

Great module!

Version ^2.0.8 throws a uknown event handler property errors in the console.

Downgrading to 1.33 fixes the issue (maybe a more recent version also works, this is the only one i've tried).

**Comment by gavinbelson** [comment]
Works all the way up to 2.0.7 actually

**Comment by rodobodolfo** [comment]
I have the same issue, switching back to 2.0.7 doesnt fix it for me:

```
Unknown event handler property onDataChange. It will be ignored.
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (created by BoardContainer)
    in BoardContainer (created by Connect(BoardContainer))
    in Connect(BoardContainer) (created by Board)
    in Provider (created by Board)
    in Board (created by KanbanBoard)`

 Unknown event handler property onCardDelete...
Unknown event handler property onCardAdd

```

**Comment by sylvain-hamel** [comment]
+1

**Comment by rodobodolfo** [comment]
I think it was fixed in 2.0.10 for me

**Comment by rcdexta** [comment]
Yes it is fixed in 2.0.10
